supervision/detection/line_zone.py

```diff
@@ -140,6 +140,13 @@ def trigger(self, detections: Detections) -> Tuple[np.ndarray, np.ndarray]:
         if len(detections) == 0:
             return crossed_in, crossed_out
 
+        if detections.tracker_id is None:
+            print(
```

Comment:
Instead of print what about extend or use our warning system 

https://github.com/roboflow/supervision/blob/2bd4348e5d05f386559ea2d526112216692b1ae2/supervision/utils/internal.py#L7

At least people can turn off instead of forceful print